src/test/java/core/FreestyleJobTest.java

```diff
@@ -16,26 +22,73 @@
 import org.jvnet.hudson.test.Issue;
 import org.openqa.selenium.By;
 import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
 
-import java.net.URL;
-import java.util.Collections;
-import java.util.concurrent.TimeUnit;
-import java.util.regex.Pattern;
-
-import static org.hamcrest.CoreMatchers.containsString;
-import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.CoreMatchers.*;
 import static org.hamcrest.Matchers.equalTo;
-import static org.jenkinsci.test.acceptance.Matchers.containsRegexp;
-import static org.jenkinsci.test.acceptance.Matchers.hasContent;
-import static org.jenkinsci.test.acceptance.Matchers.pageObjectDoesNotExist;
-import static org.jenkinsci.test.acceptance.Matchers.pageObjectExists;
-import static org.junit.Assert.assertThat;
-import static org.junit.Assert.assertTrue;
+import static org.jenkinsci.test.acceptance.Matchers.*;
+import static org.junit.Assert.*;
 
 public class FreestyleJobTest extends AbstractJUnitTest {
+    @Test
+    public void should_use_upstream_trigger() {
+        FreeStyleJob main = jenkins.jobs.create(FreeStyleJob.class);
+        FreeStyleJob trigger = jenkins.jobs.create(FreeStyleJob.class);
+
+        main.edit(() -> {
```

Comment:
`edit`? We already have `public <T> T configure(Callable<T> body)`, did not noticed when this sneaked in... Anyway, if you need different signature please stick with the name `configure`.